ext/datadog_profiling_native_extension/datadog_ruby_common.h

```diff
@@ -41,6 +41,13 @@ static inline ddog_CharSlice char_slice_from_ruby_string(VALUE string) {
   return char_slice;
 }
 
+static inline ddog_CharSlice char_slice_from_cstr(const char *cstr) {
```

Comment:
Since this is only used by `crashtracking_runtime_stacks.c`, I recommend moving it there and not adding it to the common utils.

In particular, the objective of the common utils is _code sharing_ but we need to be careful for it to not become the "misc stuff drawer" that many people have at home ;)

So once this becomes used elsewhere, we can move it here, but until then, I'd suggest avoiding.